.pre-commit-config.yaml

```diff
@@ -19,13 +19,15 @@ repos:
     hooks:
     -   id: flake8
         args: ["--ignore","E501,W503","--exclude","src/lumigo_tracer/libs/*"]
+        additional_dependencies: ['importlib-metadata==4.13.0']
 
 -   repo: https://github.com/PyCQA/bandit
     rev: '1.7.0'
     hooks:
     -   id: bandit
         exclude: ^src/test/
         args: ["-lll"]
+        additional_dependencies: ['importlib-metadata==4.13.0']
```

Comment:
fixing the cicd error `AttributeError: 'EntryPoints' object has no attribute 'items'`